test/integration/gsutil/controls/gcloud.rb

```diff
@@ -0,0 +1,72 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'retriable'
+
+control 'get_from_bucket with crcmod compilation' do
+  title "With enable_init_gsutil_crcmod_el=true and enable_get_from_bucket=true"
+
+  describe 'console output of startup-scripts-gsutil1' do
+    # Avoid racing against the instance boot sequence
+    before :all do
+      Retriable.retriable(tries: 20) do
+        get_serial_port_output = "gcloud compute instances get-serial-port-output startup-scripts-gsutil1"
+        @cmd = command("#{get_serial_port_output} --project #{attribute('project_id')} --zone #{attribute('region')}-a")
+        if not %r{systemd: Startup finished}.match(@cmd.stdout)
+          raise StandardError, "Not found: 'systemd: Startup finished' in console output, cannot proceed"
+        end
```

Comment:
@adrienthebo Spent some time this evening tracking down the test failures.  One of two issues was a race condition with the gcloud command being executed while startup was in-progress.  I'm curious on your feedback regarding this `Retriable` approach to solving the issue.  I generally avoid `before :all`, but it seems prudent in this situation.

Reply:
This does work, but waiting for a startup script to finish running feels like more of a provisioning concern rather than a testing issue. I'd expect for `kitchen converge` to block until the VMs are up and ready and that `kitchen verify` can be run. I've implemented this for the Forseti module with similar logic at https://github.com/terraform-google-modules/terraform-google-forseti/blob/master/test/fixtures/simple_example/main.tf#L57-L71 - could we shift this logic to a `null_resource` with a `local-exec` provisioner?